### PR TITLE
Feat: add cluster register,delete,update,get

### PR DESCRIPTION
**Pull request by 1ch0**

### CODE REVIEW

File: internal/biz/cluster.go [hunk 1]
```diff
@@ -97,3 +94,71 @@ func (c *ClusterUsecase) ListNamespaces(ctx context.Context, id int) ([]string,
...
 	return result, nil
 }
+
+func (c *ClusterUsecase) RegisterCluster(ctx context.Context, cluster *Cluster) (*Cluster, error) {
+	if err := c.isExisted(ctx, cluster); err != nil {
+		return nil, err
+	}
+	if _, err := c.getter.GetRepo([]byte(cluster.Kubeconfig), c.log); err != nil {
+		return nil, fmt.Errorf("get k8s repo error: %v", err)
+	}
```
Comment (mouuii) [inline internal/biz/cluster.go:104]: why add getRepo？

File: internal/biz/cluster.go [hunk 2]
```diff
@@ -97,3 +94,71 @@ func (c *ClusterUsecase) ListNamespaces(ctx context.Context, id int) ([]string,
...
+	return clusterResult, nil
+}
+
+func (c *ClusterUsecase) UpdateCluster(ctx context.Context, cluster *Cluster) (*Cluster, error) {
+	if err := c.isExisted(ctx, cluster); err != nil {
+		return nil, err
+	}
+	if _, err := c.getter.GetRepo([]byte(cluster.Kubeconfig), c.log); err != nil {
+		return nil, err
+	}
```
Comment (mouuii) [inline internal/biz/cluster.go:134]: remove getRepo

File: internal/biz/cluster.go [hunk 3]
```diff
@@ -97,3 +94,71 @@ func (c *ClusterUsecase) ListNamespaces(ctx context.Context, id int) ([]string,
...
+	if len(clusters) > 0 {
+		// check kubeconfig is existed
+		for _, v := range clusters {
+			if v.Kubeconfig == cluster.Kubeconfig {
+				return fmt.Errorf("kubeconfig is existed")
+			}
+		}
+	}
+	return nil
```
Comment (mouuii) [inline internal/biz/cluster.go:163]: check exist by id

File: internal/biz/cluster.go (hunk 1 above)
Comment (1ch0) [reply internal/biz/cluster.go:104]: Check thekubeconfig is available.

File: internal/biz/cluster.go (hunk 2 above)
Comment (1ch0) [reply internal/biz/cluster.go:134]: Check thekubeconfig is available.

File: internal/biz/cluster.go (hunk 1 above)
Comment (1ch0) [reply internal/biz/cluster.go:104]: Check the kubeconfig is available.

File: internal/biz/cluster.go (hunk 2 above)
Comment (1ch0) [reply internal/biz/cluster.go:134]: Check the kubeconfig is available.

File: internal/biz/cluster.go (hunk 3 above)
Comment (1ch0) [reply internal/biz/cluster.go:163]: What if the ids are different but the kubeconfig is the same, and the add cluster service request does not have an id.

**Pull request closed without merging**